main.go

```diff
@@ -604,3 +617,10 @@ func generatePackageDetailsHTMLElements(testSuiteOverview map[string][]string, p
 
 	return strings.Join(elem, "\n"), nil
 }
+
+func formatTimeDisplay(secs float64) (float64, string) {
+	if secs > 1 {
+		return secs, "s"
```

Comment:
Think we can perhaps truncate to 3 decimal places here. Since the time resolution we care about is only up to millisecond

Reply:
Shouldn't we truncate where the seconds/milliseconds are displayed? such as:

`"elapsedTime": fmt.Sprintf("%.3f", testCaseDetails.ElapsedTime)`

Reply:
That should also do, we just need to insure its being done everywhere where we access **ElapsedTime** rather than in a single place